pyccel/codegen/wrapper/c_to_python_wrapper.py

```diff
@@ -1481,7 +1481,7 @@ def _wrap_FunctionDef(self, expr):
             a_var = a.var
             func_scope.insert_symbol(getattr(a_var, 'original_var', a_var).name)
 
-        in_interface = len(expr.get_user_nodes(Interface)) > 0
+        in_interface = len(expr.get_user_nodes(Interface, (FunctionCall,))) > 0
```

Comment:
Why not writing the keyword parameter `excluded_nodes` explicitly as you have done below?

Reply:
Fixed : 6acee633b